tests/tasks/setup_mock_wifi_wpa3.yml

```diff
@@ -0,0 +1,105 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install packages required to set up mock wifi network
+  package:
+    name:
+      - NetworkManager
+    state: present
+
+- name: Ensure NetworkManager is running
+  service:
+    name: NetworkManager
+    state: started
+
+- name: Copy server certificates
+  copy:
+    src: "{{ item }}"
+    dest: "/etc/pki/tls/{{ item }}"
+  with_items:
+    - server.key
+    - dh.pem
+    - server.pem
+    - cacert.pem
+
+
+- name: compile hostapd from src code
+  shell: |
+    mkdir -p /etc/hostapd/
+    yum -y install wget
+    wget https://w1.fi/releases/hostapd-2.7.tar.gz --no-check-certificate
+    tar xzvf hostapd-2.7.tar.gz
+    cd hostapd-2.7/hostapd/
+    echo "CONFIG_SAE=y" >> defconfig
+    cp defconfig .config
+    yum -y install libnl3-devel
+    yum -y install openssl-devel
+    yum -y install gcc
+    make -j 2
+    cd /root/
+    wget https://w1.fi/releases/wpa_supplicant-2.7.tar.gz --no-check-certificate
+    tar xzvf wpa_supplicant-2.7.tar.gz
+    cd wpa_supplicant-2.7/wpa_supplicant/
+    echo "CONFIG_SAE=y" >> defconfig
+    cp defconfig .config
+    echo '#include "crypto/aes-omac1.c"' >> ../src/common/wpa_common.c
```

Comment:
why do you need to patch hostapd sources this way? I didn't find build errors with hostapd-2.7 and that configuration. Can't we use a copr repository with a pre-built hostapd having all the required build flags, and install the rpm before the test starts?

Reply:
maybe I need to add copr repo for RHEL-7, but for Fedora-33 and RHEL-8 rpm, it already support wpa3.